### PR TITLE
Doc page for the Hive data integration

### PR DESCRIPTION
**Pull request by Rutam21**
## Description

This PR adds a Doc page on Hive Data Integration to the MindsDB Docs website.

**Fixes** #4726 

## Type of change

- [x] 📄 This change requires a documentation update

### What is the solution?

This will add a new Info Page to the Docs website about Hive Data Integration.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.
- [x] Simple Documentation Update. The above checklist options are not applicable.

### CODE REVIEW

**Comment by Rutam21** [comment]
@martyna-mindsdb Please review and suggest any required changes. Thanks 🙂

File: docs/data-integrations/hive.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* `host`: host to server IP Address or hostname
+* `port`: port through which TCPIP connection is to be made
+* `database`: Database name to be connected
+* `auth`: Defaults to CUSTOM in case not provided. Check for other options in [https://pypi.org/project/PyHive/](https://pypi.org/project/PyHive/)
+
+
+## Usage
+In order to make use of this handler and connect to Hive in MindsDB, the following syntax can be used.
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/data-integrations/hive.mdx:26]: Please replace it with ```sql

File: docs/data-integrations/hive.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+WITH ENGINE = 'hive',             --- name of the MindsDB handler
+PARAMETERS = {
+  "user": " ",                    --- database user
+  "password": " ",                --- database password
+  "host": " ",                    --- host name or IP address
+  "port": ,                       --- port used to make TCP/IP connection
+  "database": " ",                --- database name
+  "auth": " "                     --- defaults to CUSTOM if not provided; check for options here: https://pypi.org/project/PyHive/
+};
+~~~~
```
Comment (martyna-mindsdb) [inline docs/data-integrations/hive.mdx:37]: Please replace it with ```

File: docs/data-integrations/hive.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Now, you can use this established connection to query your database as follows,
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/data-integrations/hive.mdx:41]: Please replace it with ```sql

File: docs/data-integrations/hive.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+SELECT * FROM hive_datasource.table_name;
+~~~~
```
Comment (martyna-mindsdb) [inline docs/data-integrations/hive.mdx:43]: Please replace it with ```

**Comment by martyna-mindsdb** [comment]
@Rutam21 
Please see the review comments above. Thanks.

File: docs/data-integrations/hive.mdx (hunk 1 above)
Comment (Rutam21) [reply docs/data-integrations/hive.mdx]: Changes have been accommodated. Thanks.

File: docs/data-integrations/hive.mdx (hunk 2 above)
Comment (Rutam21) [reply docs/data-integrations/hive.mdx]: Changes have been accommodated. Thanks.

File: docs/data-integrations/hive.mdx (hunk 3 above)
Comment (Rutam21) [reply docs/data-integrations/hive.mdx]: Changes have been accommodated. Thanks.

File: docs/data-integrations/hive.mdx (hunk 4 above)
Comment (Rutam21) [reply docs/data-integrations/hive.mdx]: Changes have been accommodated. Thanks.

**Comment by Rutam21** [comment]
@martyna-mindsdb All requested changes have been accommodated. Thanks.

**Pull request merged**